Packs/Ransomware/Playbooks/playbook-Post_Intrusion_Ransomware_Investigation_README.md

```diff
@@ -0,0 +1,63 @@
+Provides the first step in the investigation of ransomware attacks.
```

Comment:
Add: 
Note: This is a beta playbook, which lets you implement and test pre-release software. Since the playbook is beta, it might contain bugs. Updates to the pack during the beta phase might include non-backward compatible features. We appreciate your feedback on the quality and usability of the pack to help us identify issues, fix them, and continually improve.